docs/reference/compatibility.md

```diff
@@ -25,19 +25,15 @@ ONNX Runtime is not explicitly tested with every variation/combination of enviro
 
 * Windows
 
-  * Tested with Windows 10 and Windows Server 2019
-  * May be compatible with Windows 7+
-  * Windows Machine Learning ([Windows](../get-started/with-windows.md))
-    * CPU: Windows 8.1+
-    * GPU: Windows 10 1709+
+  * Tested with Windows 10 and Windows Server 2022. When building the official packages, the [WINVER](https://learn.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=msvc-170) macro is defined as 0x0A00 which maps to Windows 10.     
```

Comment:
@natke, our devs may wonder where the information in this page was from.  Can I put such details here? Or do you have a better place?

Reply:
@snnn I would simply say that we support Windows 10 and later